.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: c
+env:
+  global:
+  - NJOBS=2
+  - COMPILER="4.05.0"
+
+cache:
+  apt: true
+  directories:
+  - ${HOME}/opam-cache
```

Comment:
I think this does nothing of use.

Reply:
My intent was to cache anything in this directory.
According to the doc, this is what should happen...

Reply:
Yes, but doesn't the `tar czf ...` overwrite the cache? So you spend time uploading and downloading tarballs for nothing.

Reply:
Maybe I missed when the cache is stored. I would like to have a .tgz with the opam root justinitialized, and unpack it over and over.  Where should I put the line making the tar?

Reply:
The travis cache is used to preserve data between jobs. The idea is that with a configuration like Coq's with 20 jobs you only need to install opam once. It's also preserved between commits.
Currently the travis execution goes like
- download and unpack cache (if it exists)
- (on linux job) install apt packages (possibly noop if they're already there from the cache)
- do before_install (noop on linux job)
- do install, `opam init` the opam-root then archiving it in opam-cache (if there was a cache present it is destroyed)
- do script (the archive is used instead of rebuilding ocaml for every package)
- upload cache

Maybe you want to skip the install lines if ~/opam-cache/cache-$COMPILER.tgz is present?